packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/machine.test.ts

```diff
@@ -695,18 +702,6 @@ describe('Liveness Machine', () => {
       );
     });
 
-    it('should reach timeout state if disconnect event never arrives', async () => {
```

Comment:
Is there a way to unit test for the new error messages? Is that not something we were doing previously?

Reply:
whoops forgot to add that, will update!